src/invoices/Voucher/VoucherDetails.js

```diff
@@ -108,11 +110,23 @@ const VoucherDetails = ({ voucher }) => (
         />
       </Col>
     </Row>
+
+    {withVendorAddress && (
```

Comment:
in case it's `false` - it will be string `false` on the screen

Reply:
[React will ignore and skip it](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)

Reply:
right, for the `false` value. But in case it's undefined or null or 0 or empty string - it might be displayed, right?

Reply:
it's OK to keep as is if you don't expect anything apart from boolean